SlideController/Classes/TitleSlidableController.swift

```diff
@@ -85,7 +85,9 @@ class TitleSlidableController<T, N>: TitleScrollable where T: ViewSlidable, T: U
     
     func jump(index: Int, animated: Bool) {
         if isIndexValid(index) {
-            controllers[selectedIndex].isSelected = false
+            if controllers.indices.contains(selectedIndex) {
```

Comment:
Double check happens: if isIndexValid(index) and then  if controllers.indices.contains(selectedIndex)
Btw, controllers.indices.contains is better one, but you should change all places and remove isIndexValid func.